content/pt-br/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Configurando Qualidade do Serviço Para Pods
+content_type: task
+weight: 30
+update_date: 2022-07-17
+origin_version: 1.24
```

Comment:
These look unusual. I'm guessing that these are for tracking localization progress.

Has this convention been accepted by SIG Docs? (it's new to me).

Reply:
Hi, I'm new contributor on pt-br docs, so my difficults are identify pages (original version) than where used base-line for internacionalizations, in pt-br docs we have pages from v1.18, 1,19, 1.20, 1.21, 1.22, 1.23, 1.24, how can I guess what pages need update. I'ts nice to do this, make a diff in the english docs, so the pages that need update will be shown. But in pt-br we don't have versioning, so if I have a origin_version  in the docs I can use a grep to show what a version used in, and assign the contributor to ask if interested in update the docs they translated.